### PR TITLE
[FMO-268] Add Shared Library and keep old function for test purpose

### PR DESCRIPTION
**Pull request by lukaszach**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,4 +1,6 @@
-private boolean lastCommitIsBumpCommit() {
+@Library('utils') _
+
+private boolean isBumpCommit() {
```
Comment (teroxik) [inline Jenkinsfile]: this could go to the library as well as it's going to be shared right

File: Jenkinsfile [hunk 2]
```diff
@@ -30,9 +35,7 @@ pipeline {
         }
         stage('Build & Test') {
             steps {
-                sh '''
-                    sbt test
-                '''
+                sh 'sbt test'
             }
         }
         stage('Publish Snapshot') {
```
Comment (teroxik) [inline Jenkinsfile:25]: The publish step can go to the library as well, similar thing for publishing jars / docker images... so we can slim it down

File: Jenkinsfile (hunk 1 above)
Comment (lukaszach) [reply Jenkinsfile]: It's there just for a first testing iteration after the merge to develop branch. I am going to remove it after a successful publish without a loop.

File: Jenkinsfile (hunk 2 above)
Comment (lukaszach) [reply Jenkinsfile:25]: Is it always the same for all repositories? Then I can move it to the shared library.

**Review by teroxik — APPROVED** [review]
Looking good!

**Review by teroxik — COMMENTED** [review]
👍

**Pull request merged**